doc/source/whatsnew/v1.5.3.rst

```diff
@@ -37,6 +37,12 @@ Bug fixes
 
 Other
 ~~~~~
+
+.. note::
+
+    If you are using :meth:`DataFrame.to_sql`, :func:`read_sql`, :func:`read_sql_table`, or :func:`read_sql_query` with SQLAlchemy 1.4.6 or greater,
+    you may see a ``sqlalchemy.exc.RemovedIn20Warning``. These warnings can be safely ignored as pandas works toward compatibility with SQLAlchemy 2.0.
```

Comment:
Can we reword the last part? These warnings can be safely ignored for the SQLAlchemy 1.4.x releases as pandas works towards compatibility ...

Reply:
Good idea. Changed